cms/envs/devstack.py

```diff
@@ -39,7 +39,7 @@
 
 ################################# LMS INTEGRATION #############################
 
-LMS_BASE = "localhost:8000"
+# LMS_BASE = "localhost:8000"  # Edraak: Commented out because we use domain names locally.
```

Comment:
That's the thing, this one overrides the JSON value that `cms/envs/aws.py` provides. Commenting it out makes things right. Perhaps the comment is misleading. Could you please double check on your machine to see whether the `LMS_BASE` value is already set to the correct value: `courses.edraak.dw`.